test/utils/wasm_engine.hpp

```diff
@@ -26,16 +26,13 @@ class WasmEngine
 
     virtual ~WasmEngine() noexcept = default;
 
-    /// Parses input bytes and creates internal wasm module out of it.
+    /// Parses input wasm binary. The created module is discarded.
     /// Returns false on parsing error.
-    /// Consecutive invocations replace the internal module and invalidate the internal instance.
-    virtual bool parse(bytes_view input) = 0;
+    virtual bool parse(bytes_view input) const = 0;
```

Comment:
Maybe this should be `try_parse` ?

Reply:
Can be, but this API is unstable enough and not keen to do esthetic changes.